### PR TITLE
More Documentation

### PR DESCRIPTION
**Pull request by mevey**
- Removed old discussions template
- Added more details about Project Reunion

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -1,4 +1,281 @@
...
+Project Reunion APIs means we take on the work to keep these APIs working as Windows evolves 
+forward. Staying within the Project Reunion API surface ensures your app can address the 
+widest range of Windows editions and releases while using up-to-date functionality.
+
+# How we are doing this
+
+#### Working in the open
+Project Reunion relies on the power of open-source and the features of GitHub to deliver 
+code to you as fast as we can dream it up.  You should feel free to fork, create pull 
+requests, open issues, propose resolutions to issues.  You can see and influence all
```
Comment (jonwis) [inline docs/README.md:217]: "... open issues, and propose resolutions ..."

File: docs/README.md [hunk 2]
```diff
@@ -1,4 +1,281 @@
...
+
+As your app uses more Project Reunion functionality it'll be ready for additional targets 
+such as packaging, identity, AppContainer, and other future Windows Editions. Depending on 
+Project Reunion APIs means we take on the work to keep these APIs working as Windows evolves 
+forward. Staying within the Project Reunion API surface ensures your app can address the 
+widest range of Windows editions and releases while using up-to-date functionality.
+
+# How we are doing this
+
+#### Working in the open
```
Comment (jonwis) [inline docs/README.md:214]: These headers should increase by one `#` each level, so this should be `###` ... maybe make a pass through and see if there are others?

Comment (mevey) [reply docs/README.md:214]: I was trigger happy (merged before I read the comments). I made this design choice as some titles were just bigger than they needed to be. I'll make some edits in the next PR.

File: docs/README.md (hunk 1 above)
Comment (mevey) [reply docs/README.md:217]: Oxford comma :)  I will have the change in the next PR.

**Pull request merged**